encord/transformers/coco/coco_encoder.py

```diff
@@ -366,9 +366,40 @@ def get_polygon(self, object_: dict, image_id: int, size: Size) -> Union[CocoAnn
         return CocoAnnotation(area, bbox, category_id, id, image_id, iscrowd, segmentation)
 
     def get_polyline(self, object_: dict, image_id: int, size: Size) -> Union[CocoAnnotation, SuperClass]:
-        raise NotImplementedError(
-            "The get_polyline function will need to be overwritten if you want to convert these objects."
-        )
+        """Polylines are technically not supported in COCO, but here we use a trick to allow a representation."""
+        polygon = get_polygon_from_dict(object_["polyline"], size.width, size.height)
+        polyline_coordinate = self.join_polyline_from_polygon(list(chain(*polygon)))
+        segmentation = [polyline_coordinate]
+        polygon = Polygon(polygon)
+        area = polygon.area
+        x, y, x_max, y_max = polygon.bounds
+        w, h = x_max - x, y_max - y
+
+        bbox = [x, y, w, h]
+        category_id = self.get_category_id(object_)
+        id, iscrowd = self.get_coco_annotation_default_fields()
+
+        return CocoAnnotation(area, bbox, category_id, id, image_id, iscrowd, segmentation)
+
+    @staticmethod
+    def join_polyline_from_polygon(polygon: List[float]) -> List[float]:
```

Comment:
I am not sure I understand why we need this backward joining of points. Why not just write area=0? I am sure you have discussed this already. Just curious. 

I imagine that it will be a pain for ML engineers, because they will always have to remember to remove that second half of the polygon?

Reply:
I believe points in a polygon in the COCO format don't have an explicit closure i.e. they are `p0=[x0, y0, ...., xn, yn]` rather than `p1=[x0, y0, ..., xn, yn, x0, y0]`. Things like `p0` will be interpreted as closed polygons (e.g. the COCO tool plotting util does exactly that). So to define a polyline we need to define a 'closure' path that yields a line, rather than a polygon. Hence the backward path.

Reply:
The alternative would be to create a skeleton definition in the ontology for every polyline instance. This does not seem scalable.

Reply:
The truth is I am not sure how torch works. I assume that if you have an area of 0 it will just be an inconsistent state. Let me try if that helps.

Reply:
As @javi-encord mentioned, it would not be ideal and would mean we effectively constantly are changing the COCO ontology.

Reply:
And yes, also as @javi-encord said I did some tests and there is no explicit closure.